src/pybind/mgr/smb/module.py

```diff
@@ -411,6 +411,45 @@ def show(
             return resources[0].to_simplified()
         return {"resources": [r.to_simplified() for r in resources]}
 
+    @cli.SMBCommand('share update cephfs qos', perm='rw')
```

Comment:
I think it would make more sense for this method to be adjacent to other `share ...` methods, so can we move this above `show`?